src/components/common/Buttons/SocialButton.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SocialButton = props => {
+  const { className, iconName, alt } = props;
+  return (
+    <div className={`icon ${className}`}>
+      <img
+        src={require(`../../../assets/icons/${iconName}-icon.svg`)}
```

Comment:
is there any way to load an image in a react component?

This is what ESLint says doc about the require 
>Since require() does a synchronous load, it can cause performance problems when used in other locations.Further, ES6 modules mandate that import and export statements can only occur in the top level of the module’s body

Please consider [this answer](https://stackoverflow.com/a/47487760/4683950) as alternative solution